clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -499,6 +499,10 @@ public enum Command {
   BACKUP_STORE_METADATA_FROM_GRAVEYARD(
       "backup-store-metadata-from-graveyard", "Backup store metadata from graveyard in EI",
       new Arg[] { VENICE_ZOOKEEPER_URL, ZK_SSL_CONFIG_FILE, BACKUP_FOLDER }
+  ),
+  DELETE_VALUE_SCHEMA(
```

Comment:
General question, will this break superset schema? Do we need to regenerate superset schema upon removing old schemas?

Reply:
It should not, super-set schema might still contain fields from a deleted schema, but it wont be read. You might argue about optimization of the size of the super-set schema if older schema fields are dropped later, but most schema evolution are addition rather than deletion of fields.